core/cas-server-core-util/src/main/java/org/apereo/cas/util/function/ComposableFunction.java

```diff
@@ -0,0 +1,33 @@
+package org.apereo.cas.util.function;
+
+import java.util.Objects;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+/**
+ * Represents a function that accepts one argument and produces a result and
+ * could be with a Consumer of the resulting type.
+ *
+ * PLEASE REMOVE THIS IF IT'S ADDED TO THE JDK
```

Comment:
Not a javadoc thing; more of an IDE+Checkstyle warning thing. I'd rather be annoyed by INFO/WARN messages of some static analysis tool than to discover things need to be maintained by reading javadocs :)